native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/common/Util.java

```diff
@@ -195,6 +196,9 @@ private static void setOutboundRespHeaders(HttpCarbonMessage outboundResponseMsg
             outboundResponseMsg.setHeader(HttpHeaderNames.CONNECTION.toString(), Constants.CONNECTION_CLOSE);
```

Comment:
Regardless of the `keepAlive` header, we need to keep the connection alive for SSE right?

Reply:
Yes, keepAlive variable value will be set to true if the content type is event-stream. Refer native/src/main/java/io/ballerina/stdlib/http/transport/contractimpl/listener/states/SendingHeaders.java

Reply:
Then why do we have this change here: 
```bal
else if (keepAlive && (Float.valueOf(inboundReqHttpVersion) == Constants.HTTP_1_1)
        && HttpUtil.hasEventStreamContentType(outboundResponseMsg)) {
    outboundResponseMsg.setHeader(HttpHeaderNames.CONNECTION.toString(), Constants.CONNECTION_KEEP_ALIVE);
}
```

Reply:
Yeah, got your point now. Will remove the keepAlive part from the mentioned code block.